### DISCUSSION TITLE
How to not use Lidar but Realsense with the package

### DISCUSSION THREAD

**Discussion by parapara29** [hippo5329/linorobot2_hardware#9] [Q&A]
Hi everyone, I do not have a lidar to work with as of now, but an intel real sense. My initial thoughts were that I could connect my realsense camera directly to my laptop via USB cable since I am using that as a robot computer, and control the robot accordingly via wifi and ESP 32 (this is just for test, I have plans to integrate jetson later). Is it possible to work the robot this way, and how can I remove the lidar configuration then, can anyone please let me know?

**Comment by hippo5329** [comment]
Yes. It should be fine to use realsense connected to your robot computer, as realsense is supported in linorobot2. Please check linorobot2 documents. You should use serial transport instead of wifi when you have a robot computer. Serial transport is not subject to wifi interference.  A pico might work better than esp32 in this case. esp32-s3 is good though it has fewer pwm available. esp32 and pico have 16 pwm, which esp32-s3 and esp32-s2 have only 8 pwm. If you use esp32 series, you may still enable wifi even using serial transport to have the option of OTA update and remote syslog.

- Thomas
